urlchecker/core/check.py

```diff
@@ -119,12 +119,15 @@ def save_results(self, file_path, sep=",", header=None, relative_paths=True):
                 # Derive the relative path based on self.path, or relative to run
                 if relative_paths:
                     if self.path:
-                        file_name = re.sub(self.path, "", file_name).strip('/')
+                        file_name = re.sub(self.path, "", file_name).strip("/")
                     else:
                         file_name = os.path.relpath(file_name)
 
                 [writer.writerow([url, "failed", file_name]) for url in result.failed]
-                [writer.writerow([url, "white_listed", file_name]) for url in result.white_listed]
+                [
+                    writer.writerow([url, "white_listed", file_name])
+                    for url in result.white_listed
+                ]
```

Comment:
This seems to be the linter effect but there no added value here. I would say the previous style was simpler and better

Reply:
Sure, happy to change this back. I like the previous one better too.

Reply:
Fixed.